model/src/main/kotlin/OrtResult.kt

```diff
@@ -421,6 +427,10 @@ data class OrtResult(
     @JsonIgnore
     fun getExcludes(): Excludes = repository.config.excludes ?: Excludes()
 
+    private fun getLicenseFindingsCurations(id: Identifier): List<LicenseFindingCuration> =
+        if (projects.containsKey(id)) repository.config.curations?.licenseFindings.orEmpty()
+        else emptyList()
```

Comment:
Or use braces for if/else if it does not fit on a single line.

Reply:
Done.